pkgs/development/compilers/sbcl/default.nix

```diff
@@ -69,6 +70,7 @@ let
       "${sbclBootstrap}/bin/sbcl --disable-debugger --no-userinit --no-sysinit"
     else
       "${lib.getExe ecl} --norc";
+  posixUtils = if stdenv.hostPlatform.isDarwin then darwin.shell_cmds else coreutils;
```

Comment:
wow amazing find.. this raises so many more questions, I still cannot even repro the original bug in a stand-alone c program but somehow we now have a fix.

1. what is the bug
2. why does this fix it
3. what on earth is this package

but hey... 🤷  if it works it works.